src/main/java/com/textmessenger/controller/MessageController.java

```diff
@@ -2,6 +2,7 @@
 
 import com.textmessenger.model.entity.Dialog;
 import com.textmessenger.model.entity.Message;
+import com.textmessenger.model.entity.User;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.textmessenger.model.entity.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=120&issues=AWUz3g7-W2AOP0CkBtXZ&open=AWUz3g7-W2AOP0CkBtXZ)

[ISSUE_KEY]: AWUz3g7-W2AOP0CkBtXZ